src/components/common/ProductTag.tsx

```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components';
+
+interface ProductTagContainerProps {
+  width: number;
+  color: string;
+  children: string;
+}
+
+interface ProductTagProps {
+  children: string;
+}
+
+const ProductTag = (props: ProductTagProps) => {
+  const { children } = props;
+  let width, color;
+
+  switch (children) {
+    case 'BEST':
+      width = 3.6;
+      color = 'pink_100';
+      break;
+
+    case '단독':
+      width = 3;
+      color = 'green_300';
+      break;
+
+    case '오늘드림':
+      width = 4.6;
+      color = 'sky_300';
+      break;
+
+    case '증정':
+      width = 3;
+      color = 'red_300';
+      break;
+
+    default:
+      width = 3.6;
+      color = 'pink_100';
+      break;
+  }
+
+  return (
+    <St.ProductTagContainer width={width} color={color}>
+      {children}
+    </St.ProductTagContainer>
+  );
+};
+
+export default ProductTag;
+
+const St = {
+  ProductTagContainer: styled.span<ProductTagContainerProps>`
```

Comment:
호옥시나 해서 요기 props 여러개여도 꼭 interface 설정하지 않고 이렇게 사용하는 것도 당연히 가능하다는 점만 쑝 짚고가요!! 😆
```suggestion
  ProductTagContainer: styled.span<{width: number, height: string}>`
```

Reply:
허걱 고마워! 이렇게 하는게 더 나으려나..? @_@

Reply:
요건 언니가 보기에 더 가독성 좋다거나 편하다고 생각하는 대로 하면 댈 것 가타 ㅎㅎ